File: extract_blueprint.js
const fs = require('fs');
const path = require('path');

const logPath = 'C:\\Users\\Guru\\.gemini\\antigravity\\brain\\57b80c0f-84e4-4450-835e-ffd6a6124dee\\.system_generated\\logs\\overview.txt';

try {
    const data = fs.readFileSync(logPath, 'utf8');
    const lines = data.split('\n');
    const jsonLine = JSON.parse(lines[6]);
    const content = jsonLine.content;
    
    fs.writeFileSync('blueprint_full.md', content, 'utf8');
    console.log('Blueprint extracted successfully');
} catch (err) {
    console.error('Error:', err);
}

File: extract_blueprint.py
import json
import os

log_path = r'C:\Users\Guru\.gemini\antigravity\brain\57b80c0f-84e4-4450-835e-ffd6a6124dee\.system_generated\logs\overview.txt'

with open(log_path, 'r', encoding='utf-8') as f:
    lines = f.readlines()
    # Line 7 (index 6) is the one we want
    data = json.loads(lines[6])
    content = data.get('content', '')
    
    with open('blueprint_full.md', 'w', encoding='utf-8') as out:
        out.write(content)
